cmd/libs/go2idl/client-gen/main.go

```diff
@@ -105,12 +106,12 @@ func main() {
 
 	if *test {
 		arguments.InputDirs = append(dependencies, []string{
-			"k8s.io/kubernetes/cmd/libs/go2idl/client-gen/testdata/apis/testgroup.k8s.io",
+			"k8s.io/kubernetes/cmd/libs/go2idl/client-gen/test_apis/testgroup.k8s.io",
```

Comment:
"testdata" is a magic word to Go, and any dir named "testdata" is ignored
by the tools.  Apparently this includes not looking at vendor/ dirs.
 ugorji calls `go run` on a file in this dir, thus it fails to compile.

On Thu, May 5, 2016 at 11:20 PM, Chao Xu <notifications@github.com> wrote:

> In cmd/libs/go2idl/client-gen/main.go
> <https://github.com/kubernetes/kubernetes/pull/24242#discussion_r62292673>
> :
>
> > @@ -105,12 +106,12 @@ func main() {
> >
> >  	if *test {
> >  		arguments.InputDirs = append(dependencies, []string{
> > -			"k8s.io/kubernetes/cmd/libs/go2idl/client-gen/testdata/apis/testgroup.k8s.io",
> > +			"k8s.io/kubernetes/cmd/libs/go2idl/client-gen/test_apis/testgroup.k8s.io",
>
> Do you know why ugorji is picky on the folder name?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24242/files/9bd7abdc6eff192782c250b5dcfbe486eeabc471..b2a5b40281484bef68f78271d744be7efeae9638#r62292673>
>